packages/pages/src/dev/server/ssr/pageLoader.ts

```diff
@@ -84,6 +84,10 @@ export const pageLoader = async ({
       locale,
       projectStructure
     );
+    if (document === null) {
+      // Fall back to localData
+      document = await getLocalDataForEntity(entityId, locale);
```

Comment:
Idk if we want to do this. Can you verify in the discuss-react channel what the functionality should be? It could be a little strange where you have local data for an entity that no longer exists and you're dev'ing against it only to find in prod that it's not there. Sort of and edge case and this fallback could be useful. Just not sure so let's see what Product wants.

Reply:
Updating to not fall back and just let it bomb out